test/base/EscrowFactoryConstructorTest.js

```diff
@@ -0,0 +1,32 @@
+// JS Libraries
+const withData = require('leche').withData;
+const { t } = require('../utils/consts');
+
+// Smart contracts
+const Escrow = artifacts.require("./base/Escrow.sol");
+const EscrowFactory = artifacts.require("./base/EscrowFactory.sol");
+
+contract('EscrowFactoryConstructorTest', async function (accounts) {
+    withData({
+        _1_basic: [undefined, false],
+    }, function(expectedErrorMessage, mustFail) {
+        it(t('user', 'new', 'Should be able to create a new instance.', mustFail), async function() {
+            // Setup
+
+            try {
+                // Invocation
+                const escrow = await Escrow.new();
```

Comment:
In this case if you are testing the factory constructor, the line ```const escrow = await Escrow.new();``` should be in the setup section (line 15).

Reply:
done